chrome/locale/en-US/zotero/preferences.ftl

```diff
@@ -71,6 +71,10 @@ preferences-styleManager-add-button =
 
 preferences-advanced-enable-local-api =
     .label = Allow other applications on this computer to communicate with { -app-name }
+preferences-advanced-local-api-running = Running on <code data-l10n-name="url">{ $url }</span>
```

Comment:
"Available at"? "Running at"? "Running on" makes sense for the host/port, but maybe not with a full URL?